pkg/execute/executor.go

```diff
@@ -11,6 +11,7 @@ import (
 	"gopkg.in/yaml.v3"
```

Comment:
It would be great to modify the E2E test - add an enabled executor definition, which is not bound to the test channel.